src/main/java/com/epam/prejap/teatrees/game/Printer.java

```diff
@@ -29,4 +39,35 @@ void endRow() {
     void border(int width) {
         out.println("+" + "-".repeat(width) + "+");
     }
+
+    /**
+     * Prints the concluding message about achieved score. In case of new record prints additional message.
+     *
+     * @param isNewBestRecord indicates whether new score is the best yet for the Player
+     * @param score           user's score, got during the game
+     * @param nameOfThePlayer
+     */
+    public void printFinalMessage(List<Record> scores, boolean isNewBestRecord, int score, String nameOfThePlayer) {
+        scores.forEach(out::println);
+
+        out.println("\nScore: " + score);
+        if (isNewBestRecord) {
+            out.println("Congrats! It's your new record " + nameOfThePlayer + "!");
+        }
+    }
+
+    /**
+     * Asks Player for his name that can be only 3 characters long.
+     *
+     * @param scanner and object, responsible for scanning Player's output
+     */
+    public String askForUserName(Scanner scanner) {
```

Comment:
This violates SRP. Printer shouldn't be responsible for reading. Consider creating a new class for it.